examples/poaps/backend/src/methods/fetch_multiple_poaps.ts

```diff
@@ -0,0 +1,17 @@
+import { POAP, PoapsClient } from '@poap-xyz/poaps';
+import { PaginatedResult } from '@poap-xyz/utils';
+
+export const fetch_multiple_poaps = async (
+  client: PoapsClient,
+): Promise<void> => {
+  try {
+    const data: PaginatedResult<POAP> = await client.fetch({
+      order: 'asc',
+      limit: 10,
+      offset: 1,
+    });
+    console.log(data);
```

Comment:
What does this return? Maybe adding a comment saying what is to be expected from the response. Something like:

> The first 10 POAP tokens minted.